html-forms/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Form</title>
+</head>
+<body>
+  <form action="" method="get">
+    <fieldset>
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <input type="text" name="email" id="email">
+      <div>
+        <label for="dropdown">What is coding cool?</label>
+      </div>
+      <select id="dropdown" name="dropdown">
+        <option value="option">Select an option</option>
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+      <div>
+        <label for="language">Favorite Language (choose one)</label>
```

Comment:
instead of a label here, I would put the text directly in a div, and wrap a label around each of the inputs.